src/visualization/types/SingleStatWithLine/view.tsx

```diff
@@ -147,92 +149,161 @@ const SingleStatWithLine: FC<Props> = ({properties, result, timeRange}) => {
     return <EmptyGraphMessage message={INVALID_DATA_COPY} />
   }
 
-  return (
-    <Plot
-      config={{
-        ...currentTheme,
-        table: result.table,
-        xAxisLabel: properties.axes.x.label,
-        yAxisLabel: properties.axes.y.label,
-        xDomain,
-        onSetXDomain,
-        onResetXDomain,
-        yDomain,
-        onSetYDomain,
-        onResetYDomain,
-        ...axisTicksOptions,
-        legendColumns,
-        legendOpacity: tooltipOpacity,
-        legendOrientationThreshold: tooltipOrientationThreshold,
-        legendColorizeRows: tooltipColorize,
-        valueFormatters: {
-          [xColumn]: xFormatter,
-          [yColumn]: yFormatter,
-        },
-        layers: [
-          {
-            type: 'line',
-            x: xColumn,
-            y: yColumn,
-            fill: groupKey,
-            interpolation,
-            position: properties.position,
-            colors: colorHexes,
-            shadeBelow: !!properties.shadeBelow,
-            shadeBelowOpacity: 0.08,
-            hoverDimension: properties.hoverDimension,
+  const latestValues = getLatestValues(result.table)
```

Comment:
Similar to my comments on https://github.com/influxdata/ui/pull/830
`getLatestValues` should exported in Giraffe. `formatStatValue` should also be exported. Release a new version of Giraffe and then consume it here.